src/commands/git_secret_tell.sh

```diff
@@ -21,13 +21,16 @@ function tell {
   local emails
   local self_email=0
   local homedir
+  local verbose=0
 
   # A POSIX variable
   # Reset in case getopts has been used previously in the shell.
   OPTIND=1
 
-  while getopts "hmd:" opt; do
+  while getopts "vhmd:" opt; do
     case "$opt" in
+      v) verbose=1;;
```

Comment:
Maybe we can make `-v` a global option? Something like:

```bash
git secret -v tell user@email.com
```

or 

```bash
GIT_SECRET_VERBOSE=1 git secret tell some@random.name 
```

Reply:
We could.

Also presumably since it would also effect `clean` and `hide`'s -v options too, I suggest we implement GIT_SECRET_VERBOSE (if we decide to do it) in a separate PR. Discussion moved to #323